package.json

```diff
@@ -28,7 +28,7 @@
     "koa-route": "^2.4.0",
     "koa-statsd": "^0.0.2",
     "moment": "^2.8.4",
-    "mongoose": "3.8.23",
+    "mongoose": "^4.0.1",
```

Comment:
Yeah I think it should be fine to with the switch to semver. But, to be honest I'm not so keen on using carets as default as we have been doing up  until now. It seems we may run into a few more issues. I'l bring it up on hipchat.